bin/post_compile

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+pip install -r requirements-dev.txt
```

Comment:
I'm not sure if Heroku won't run that in the production environment.

Reply:
good spot!

i wonder how we can do this only on staging